engines/bastion_katello/app/assets/javascripts/bastion_katello/products/details/repositories/new/new-repository.controller.js

```diff
@@ -141,10 +141,8 @@ angular.module('Bastion.repositories').controller('NewRepositoryController',
 
             $scope.collectionURLPopover = $sce.trustAsHtml("You can sync collections utilizing just the url:<br/>" +
           "<b>1. For all collections in Ansible Galaxy:</b><br/>" +
-          "https://galaxy.ansible.com/api/v2/collections <br/>" +
-          "<b>2. For specific collection with URL filtering:</b><br/>" +
-          "https://galaxy.ansible.com/api/v2/collections/testing/k8s_demo_collection <br/>" +
-          "<b>3. For specific collections with Requirements.yml:</b><br/>" +
+          "https://galaxy.ansible.com/api/<br/>" +
```

Comment:
Think https://galaxy.ansible.com/ is used for mirroring although the last I tried to mirror, it failed on some collection in galaxy..Not sure we want to advertise mirroring?

Reply:
@sjha4 you mean mirroring all of Galaxy? Do we know why this was presented as a tool tip in the first place?

Reply:
Ya..That was supposed to be a use case for collections support..Mirroring all of galaxy. I don't mind leaving that as advertised. Only issue was last I tried it, it failed on some collection in the galaxy. Think we'll have to test it better so it doesn't stay broken.

Reply:
@sjha4 @jlsherrill aside from the comments re the galaxy url being listed... any other items we need to adjust? I think @jeremylenz is waiting on this for 4.1 picks.

Reply:
maybe we should discuss it more with the pulp-ansible team and merge this as is.  ACK from me

Reply:
Ack +1